eng/performance/scenarios.yml

```diff
@@ -22,6 +22,12 @@ jobs:
         displayName: '${{ parameters.osName }} ${{ parameters.osVersion }} ${{ parameters.architecture }} ${{ parameters.kind }} ${{ parameters.machinePool }}'
         timeoutInMinutes: 320
         variables:
+        - ${{ if ne(parameters.runEnvVars, '')}}:
```

Comment:
When you change the default to `[]`, you can replace this condition with a `ne(length(parameters.runEnvVars), 0)`